Resources/Prototypes/Entities/Objects/Specific/Security/barrier.yml

```diff
@@ -0,0 +1,53 @@
+- type: entity
+  name: deployable barrier
+  id: DeployableBarrier
+  description: A deployable barrier. Swipe your ID card to lock/unlock it.
+  parent: BaseStructure
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Security/barrier.rsi
+    state: idle
+  - type: InteractionOutline
+  - type: Physics
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+          bounds: "-0.5, -0.5, 0.5, 0.5"
+      layer:
+      - Opaque
+      - Impassable
+      - MobImpassable
+      - VaultImpassable
+      - SmallImpassable
```

Comment:
```suggestion
      layer:
      - Opaque
      - Impassable
      - VaultImpassable
      - SmallImpassable
      mask:
      - Impassable
```

Reply:
@SweptWasTaken @metalgearsloth after this change you can't pull barriers through airlocks - its getting stuck. And why you have removed MobImpassable layer?

Reply:
> after this change you can't pull barriers through airlocks - its getting stuck

It had no collision at all previously so was just going through walls, you need to fix the bounds.

And I don't think barriers should care about whether something is a mob (and it's probably being removed in the eventual layer refactor).

Reply:
![image](https://user-images.githubusercontent.com/31366439/130360042-7e7a9d22-943b-4ffd-b452-e1b882119da4.png)

Reply:
I moved it to circle shape, works like a charm with 0.45 radius